parser/context.go

```diff
@@ -249,7 +249,14 @@ func specialChildren(node ast.Node) []ast.Node {
 	case *ast.Slice:
 		return nil
 	case *ast.Local:
-		return nil
+		if len(node.Binds) == 0 {
```

Comment:
Is it necessary? Especially if we add `body` (see comment below), but generally it seems that:
a) That's impossible - at least one binding is required 
b) I think we could just make a 0-length slice using the code below

Reply:
done